### PR TITLE
(docs) Correct metadata for url, os and versions

### PR DESCRIPTION
**Pull request by cyberious**
- Was missing pe and puppet versions
- Was missing osfamily windows and versions
- Correct license to Apache-2.0
- Add tags for searchability

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,16 +1,44 @@
...
-  "license": "Apache License, Version 2.0",
+  "license": "Apache-2.0",
   "source": "https://github.com/puppetlabs/puppetlabs-dism",
   "project_page": "https://github.com/puppetlabs/puppetlabs-dism",
-  "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
-  "types": [
-  
+  "issues_url": "https://tickets.puppetlabs.com/browse/MODULES/component/12014",
+  "tags": [
+    "Deployment-Image-Servicing-and-Management",
```
Comment (ferventcoder) [inline metadata.json]: uppercase ?

**Comment by ferventcoder** [comment]
LGTM - the question on uppercase tho.

File: metadata.json [hunk 2]
```diff
@@ -1,16 +1,44 @@
...
+    "dism",
+    "feature",
+    "microsoft",
+    "iis",
```
Comment (ferventcoder) [inline metadata.json]: What does IIS have to do with this one?

Comment (ferventcoder) [inline metadata.json]: I get you can install IIS, but that is an artifact of using dism, not a feature. Am I missing something important?

**Comment by cyberious** [comment]
updated @ferventcoder

**Comment by cyberious** [comment]
Uppercase was copy pasta from dism documentation and didn't lowercase it

**Pull request merged**